internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -44,11 +46,18 @@
 {{- end}}
 {{- end}} {{/* end if eq .WorkloadType "Load Balanced Web Service"*/}}
 {{- if eq .WorkloadType "Backend Service"}}
-{{- if eq .HTTPTargetContainer.Name .WorkloadName}} 
+{{- if eq .HTTPTargetContainer.Name .WorkloadName}}
```

Comment:
Great question: It looks like the code is saying that we expose the port via PortMappings for a backend service only if the target container is the main container. But in fact, the previous code was saying that if the target container is main container then add new field `Name` in the `portmapping` because SC will use that field for the communication purpose otherwise just use portmapping without `Name` field in it. So I just kept that condition as is for now because with SC feature release this  `{{- if eq .HTTPTargetContainer.Name .WorkloadName}}` condition will go away and we will be able to set portmappings irrespective of anything. Does that make sense? I know it is little confusing but happy to explain offline as well.